introduction_to_amazon_algorithms/jumpstart-foundation-models/code-llama-fine-tuning-evaluate-human-eval.ipynb

```diff
@@ -0,0 +1,1284 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    hyperparameters.validate(
</pre><p>non-blocking: It would be much cleaner to just use the <code>hyperparameters={"epoch": "1"}</code> in the initializer for JumpStartEstimator rather than retrieving, updating, and validating externally.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>